### ISSUE TITLE
1.4 dark preset

### ISSUE THREAD

**Issue by Secret-chest** [ScratchAddons/ScratchAddons#2244]
Please add a 1.4-inspired dark preset that I made:
![image](https://user-images.githubusercontent.com/74449186/115538028-20967680-a2a4-11eb-9d64-0b15b2ee69b3.png)
Colors:
Page bg: 8C8C8C
Highlight color: 3E9ECB
Menu bar: 8C8C8C
Active tab bg: 626E75
Inactive tab bg: 59596A
Sprite list bg: 6A6A6A
Asset list bg: 6A6A6A
Selected list item bg: 404143
Accent bg: 5E686C
Input bg: 55585B
Workspace bg: 595D5E
Block category menu bg: 666B77
Block pallete bg: 606369
Full screen bg: 000000
Full screen header bg: 000000
Border color: FFFFFF with 5%
Dots: No

**Comment by Secret-chest** [comment]
![image](https://user-images.githubusercontent.com/74449186/115539191-56882a80-a2a5-11eb-9b72-ecab3810a490.png)

**Comment by mxmou** [comment]
This looks really good, but the text on the selected tab doesn't have very good contrast. I think I should add a setting that could change it.

**Comment by WorldLanguages** [comment]
Related: #2232

At some point we'll probably have many presets as users submit more, so a dedicated part of the setting page for editor themes based on the `editor-dark-mode` addon would be interesting :thinking:

**Comment by mxmou** [comment]
Alternatively, we could add a way for themes to be mutually exclusive and add something like this (incomplete, just a concept):
![image](https://user-images.githubusercontent.com/51849865/115548799-2beba100-a2a8-11eb-9199-dcccb91535cd.png)

**Comment by WorldLanguages** [comment]
@mxmou Yep, these new "themes" would be able to set presets on multiple addons, enable/disable them etc. In case you work on a 2.0 addon or 1.4 addon. (And ofc 2.0 theme or whatever would turn on `editor-stage-left`)
